packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactEditText.java

```diff
@@ -238,6 +239,12 @@ public boolean isLayoutRequested() {
   @Override
   protected void onLayout(boolean changed, int left, int top, int right, int bottom) {
     onContentSizeChange();
+    if (mSelectTextOnFocus) {
+      // Explicitly call this method to select text when layout is drawn
+      selectAll();
+      // Prevent text on being selected for next layout pass
+      mSelectTextOnFocus = false;
+    }
```

Comment:
Shouldn't we only select text when the text is actually focused?

Making the text selected even when not focused doesn't seem to be right by this propname.